python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/metadata.schema.yml

```
@@ -381,3 +381,9 @@ keys:
           password:
             type: str
             description: Local password assigned to a replica of the fabric device within the Digital Twin environment.
+          act_internet_access:
+            type: bool
+            description: |-
+              Specifies if the ACT Digital Twin device is deployed with direct access to the Internet.
+              This option applies only to the 'cloudeos' and 'veos' node types and will be ignored for all other ACT node types.
+              ACT does not provide direct Internet access to cloudeos or veos devices by default.
```

I think backticks render properly in the doc:
```suggestion
              Specifies if the ACT Digital Twin device is deployed with direct access to the Internet.
              This option applies only to the `cloudeos` and `veos` node types and will be ignored for all other ACT node types.
              ACT does not provide direct Internet access to `cloudeos` or `veos` devices by default.
```

---

You can change the other schema descriptions too if that works.

---

Fixed (changed to backticks)